### ISSUE TITLE
How to use bdxt

### ISSUE THREAD

**Issue by mhaboali** [ZHI-ANG/LeGO-LOAM-GPS#1]
When I tried to build your pkg, I got this error:

```
CMake Error at /opt/ros/melodic/share/catkin/cmake/catkinConfig.cmake:83 (find_package):
  Could not find a package configuration file provided by "bdxt" with any of
  the following names:

    bdxtConfig.cmake
    bdxt-config.cmake

  Add the installation prefix of "bdxt" to CMAKE_PREFIX_PATH or set
  "bdxt_DIR" to a directory containing one of the above files.  If "bdxt"
  provides a separate development package or SDK, be sure it has been
  installed.
Call Stack (most recent call first):
  CMakeLists.txt:6 (find_package)
```
Thanks!

**Comment by kunnalparihar** [comment]
@mhaboali @ZHI-ANG did you find any answer for the above issue?

**Comment by mhaboali** [comment]
No, unfortunately, I'd not figured it out and that blocked me of using this library.

**Comment by ZHI-ANG** [comment]
@mhaboali @kunnalparihar I'm sorry that I give no reply until now. The package "bdxt" was just a driver package we wrote for an IMU. "bdxt" collects the robot pose RPY and sends it to the function "rtkHandler()" in transformFusion.cpp. You can write your own IMU driver only if adapting to the "nav_msgs::Odometry" data structure.

By the way, this repository was for testing coordinate transformation in LeGO-LOAM during my internship. I intended to set the  global init pose for the robot, so that it could accumulate pose calculation from its GPS pose instead of (0,0,0). However, It turned out that the transformation was ok but appeared in a wrong place. LeGO-LOAM init its coordinate in somewhere else. GPS pose here won't be taken as the initial pose into LeGO-LOAM. Then my internship was over and I did not found a better solution to this problem.
